lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -60,6 +65,14 @@ def process_request(self, request):
         if not CoursePreference.course_allows_nonregistered_access(course_id):
             return None
 
+        if not modulestore().has_course(course_id):
+            log.warning(
+                u"Anonymous user tried to sneakpeek non-existent course %s with lingering preferences",
```

Comment:
This line looks too long.  Lines must be below 80 characters.  

You can put the text in a 'msg' variable or extend the text line with a "\"